content/posts/newsletter-december-2024.md

```diff
@@ -0,0 +1,32 @@
+title: Read the Docs Newsletter - December 2024
+date: December 11, 2024
+description: Company updates and new features from the last month, current focus, and upcoming features.
+category: Newsletter
+tags: newsletter, community
+authors: Eric Holscher
+status: published
+image: /images/december-2024.jpg
+image_credit: Photo by <a href="https://unsplash.com/@thebeardbe?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Filip Bunkens</a> on <a href="https://unsplash.com/photos/road-covered-by-snow-near-vehicle-traveling-at-daytime-R5SrmZPoO40?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Unsplash</a>
+
+## News and updates
```

Comment:
We can add here:

* Added support for long environment variables: https://github.com/readthedocs/readthedocs.org/pull/11748
* All the build tools were updated: https://github.com/readthedocs/readthedocs.org/pull/11742